UPGRADE-3.4.md

```diff
@@ -234,6 +234,27 @@ Translation
    and will be removed in 4.0, use `Symfony\Component\Translation\Writer\TranslationWriter::write` 
    instead. 
 
+ * Passing a `Symfony\Component\Translation\MessageSelector` to `Translator` has been
+   deprecated. You should pass a message formatter instead
+
+  Before:
```

Comment:
Just noticed: This line and the following need to be indented by an additional space so that it will be rendered property as part of the list item.

Reply:
fixed